Add missing tutorials (#288)
flairNLP/fundus: docs/1_getting_started.md

# Basics

This tutorial explains the basic concepts of Fundus:

> - What is the `PublisherCollection`

Reviewer: ```suggestion
- `PublisherCollection`
```

> - What is a `Crawler`

Reviewer: ```suggestion
- `Crawler`
```

- How to retrieve articles

## `PublisherCollection`

> Fundus comes with a collection of publisher-specific parsers grouped by country of origin.

Reviewer: ```suggestion
Fundus includes a collection of publisher-specific parsers grouped by country of origin.
```

You can access these publishers through a single class called `PublisherCollection` using the [Alpha-2](https://www.iban.com/country-codes) codes described in ISO 3166 as identifiers.

> The `PublisherCollection` works as an information hub describing publishers [currently supported](supported_publishers.md) by Fundus.

Reviewer: ```suggestion
```

Author: Changed this line a bit so it's still referencing the supported table.

> For example, to access all publishers based in the US simply write:

Reviewer: ```suggestion
To access all publishers based in the US:
```

````python
from fundus import PublisherCollection

PublisherCollection.us
````

## The crawler

> If you want to crawl articles, you need to initialize a crawler first.

Reviewer: ```suggestion
If you want to crawl articles, the first step is to initialize a crawler.
```

You can do so by using the publisher collection (recommended) or low-level objects.

> As an example, let's initiate a crawler capable of crawling publishers based in the US by utilizing the `PublisherCollection`

Reviewer: ```suggestion
Let's initiate a crawler capable of crawling publishers based in the US by utilizing
the `PublisherCollection`
```
